pkg/rpcclient/wsclient.go

```diff
@@ -804,11 +804,11 @@ func (c *WSClient) ReceiveExecutions(flt *neorpc.ExecutionFilter, rcvr chan<- *s
 }
 
 // ReceiveNotaryRequests registers provided channel as a receiver for notary request
-// payload addition or removal events. Events can be filtered by the given TxFilter
+// payload addition or removal events. Events can be filtered by the given NotaryRequestFilter
 // where sender corresponds to notary request sender (the second fallback transaction
 // signer) and signer corresponds to main transaction signers. nil value doesn't add
```

Comment:
Also a couple of words about `type` filter as far, something like "type corresponds to the mempool event type and denotes whether notary request was added to or removed from the notary request pool".

Reply:
added